tldw_Server_API/app/core/DB_Management/chacha/conversation_store.py

```
@@ -0,0 +1,1322 @@
+                "INSERT INTO conversation_settings(conversation_id, settings_json, last_modified) "
+                "VALUES (%s, %s, NOW()) "
+                "ON CONFLICT (conversation_id) DO UPDATE SET settings_json = EXCLUDED.settings_json, "
+                "last_modified = NOW()"
+            )
+            self._db.backend.execute(upsert, (conversation_id, payload))
+            self._db.backend.execute(
+                "UPDATE conversations SET version = version + 1, last_modified = NOW() "
+                "WHERE id = %s AND deleted = 0",
+                (conversation_id,),
+            )
+            return True
```

Ruff (0.15.10)

[warning] Consider moving this statement to an `else` block

(TRY300)